package.json

```diff
@@ -12,10 +12,11 @@
   "keywords": [],
   "main": "index.js",
   "scripts": {
-    "dev": "node ./node_modules/nodemon/bin/nodemon.js ./server.js",
+    "dev": "npm run build:tailwind && node ./node_modules/nodemon/bin/nodemon.js ./server.js -e ejs,js,css,html",
```

Comment:
@rupl I'm running `docker-compose up` locally and I can see it runs the start script. Should I be running the `dev` script locally? Then I could avoid adding the `NODE_ENV=production` in `start`?
To be discusses whenever. For now, this all works!

Reply:
I've opened a ticket to work on a fix for this: https://humanitarian.atlassian.net/browse/OPS-7176